src/components/ResultsBody/Wrapper/index.jsx

```diff
@@ -0,0 +1,72 @@
+import styles from "./wrapper.module.css";
+import {
+  Autocomplete,
+  Box,
+  Button,
+  Icon,
+  Link,
+  MenuItem,
+  TextField,
+} from "@mui/material";
+import { PaginatedList } from "../PaginatedList/index.jsx";
+import React, { useState } from "react";
+import Typography from "@mui/material/Typography";
+
+export function Wrapper({ arrayOfSMLinks }) {
+  const arrayOfPaginationSettings = [6, 18, 36];
+  const [numberOfCards, setNumberOfCards] = useState(6);
+
+  const handleNumberOfCardsChange = (event) => {
+    setNumberOfCards(event.target.value);
+  };
+
+  return (
+    <div className={styles.wrapper}>
+      <div className={styles.toolbar}>
+        <div className={styles.paginationSettingsWrapper}>
+          <Box>show</Box>
+          <Box className={styles.textFieldBox}>
+            <div>
+              <TextField
+                id="select-number-of-cards"
+                select
+                value={numberOfCards}
+                onChange={handleNumberOfCardsChange}
+                className={styles.textField}
+                variant="outlined"
+                SelectProps={{ IconComponent: () => null }}
```

Comment:
WTF, you are passing component which is not rendering anything

Reply:
done